internal/testutil/acc/factory.go

```diff
@@ -51,6 +51,17 @@ func ConnV2UsingProxy(proxyPort *int) *admin.APIClient {
 	return client.(*config.MongoDBClient).AtlasV2
 }
 
+func ConnV2UsingGov() *admin.APIClient {
+	cfg := config.Config{
+		PublicKey:    os.Getenv("MONGODB_ATLAS_GOV_PUBLIC_KEY"),
+		PrivateKey:   os.Getenv("MONGODB_ATLAS_GOV_PRIVATE_KEY"),
+		BaseURL:      os.Getenv("MONGODB_ATLAS_GOV_BASE_URL"),
+		RealmBaseURL: os.Getenv("MONGODB_ATLAS_GOV_BASE_URL"),
```

Comment:
I believe realm uses a different URL, meaning `BaseURL` != `RealmBaseURL`. Would simply leave the realm url undefined and we can configure this if we have the need in the future.

Reply:
I believe Agustin is right, the URL is not the same. Also agree on the action

Reply:
Changed ✅